client/my-sites/plans-features-main/index.tsx

```diff
@@ -722,6 +728,48 @@ const PlansFeaturesMain = ( {
 		[]
 	);
 
+	const handlePlanTypeSelectorChange = useCallback(
+		( selectedItem: { key: SupportedUrlFriendlyTermType } ) => {
```

Comment:
**Note:**

@jdc91 Looking for a confidence check. I copied L733-749 from the `useIntervalOptionHook` which you wrote. There is an `addtionalPathProps` value that reads the `redirectTo` prop from `planTypeSelectorProps`. From what I can tell, the value never exists on `planTypeSelectorProps`. Could you elaborate on what's happening?

Could you

Reply:
I'll do some auditing

Reply:
Looks to be unused. Removing references to it